### ISSUE TITLE
How to customize custom template path ?

### ISSUE THREAD

**Issue by rbecheras** [update/updater-gitignore#1]
How to use custom templates directory instead of the templates dir from the user home when using it as a CLI ?

And if I use `updater-gitignore` as a plugin, how to do the same thing ? Typically using the `templates` directory of the current updater source directory.

**Comment by jonschlinkert** [comment]
You should be able to use the `~/templates` directory. Templates for dotfiles always start with `_`, so the `.gitignore` template would be saved at `~/templates/_gitignore`.

I think there is another issue somewhere where we discussed changing that directory to `~/{app}/templates/`, where `app` is `assemble`, `verb`, `update`, `generate` etc.  We also discussed supporting `~/toolkit/templates` as well, where each app would look in the app templates directory first, then fallback to the toolkit one.  

thoughts?

**Comment by rbecheras** [comment]
Yes, using conventions could be interesting but allowing to use a fully customized path should be even more.

I search to create updaters that use other updaters as plugins for the imperative work, but using their own  templates (from the parent updater's `./templates` directory).

It is already possible out of the field of this issue ?
